src/main/java/cam72cam/immersiverailroading/entity/EntityRollingStock.java

```diff
@@ -42,7 +43,7 @@
 
 public abstract class EntityRollingStock extends Entity implements IEntityAdditionalSpawnData {
 	
-	private static DataParameter<Integer> LOCK_TYPE = EntityDataManager.createKey(EntityRollingStock.class, DataSerializers.VARINT);	//0=everyone can enter/break, 1=everyone can enter, 2=no one can enter/break
+	private static DataParameter<String> LOCK_TYPE = EntityDataManager.createKey(EntityRollingStock.class, DataSerializers.STRING);	//0=everyone can enter/break, 1=everyone can enter, 2=no one can enter/break
```

Comment:
This should remain an integer synchronizer.  Use the ordinal value of the enum instead